docs/admin/node-allocatable.md

```diff
@@ -9,115 +9,182 @@ title: Reserving Compute Resources for System Daemons
+for more details on recommended control group hierarchy.
+
+Note that Kubelet **does not** create `--kube-reserved-cgroup` if it doesn't
+exist. Kubelet will fail if an invalid cgroup is specified.
 
 ### System Reserved
 
 - **Kubelet Flag**: `--system-reserved=[cpu=100mi][,][memory=100Mi]`
 - **Kubelet Flag**: `--system-reserved-cgroup=`
 
 
-`system-reserved` is meant to capture resource reservation for OS system daemons like `sshd`, `udev`, etc.
-`system-reserved` should reserve `memory` for the `kernel` too since `kernel` memory is not accounted to pods (yet) in Kubernetes.
-Reserving resources for user login sessions is also recommended (`user.slice` in systemd world).
+`system-reserved` is meant to capture resource reservation for OS system daemons
+like `sshd`, `udev`, etc. `system-reserved` should reserve `memory` for the
+`kernel` too since `kernel` memory is not accounted to pods (yet) in Kubernetes.
+Reserving resources for user login sessions is also recommended (`user.slice` in
+systemd world).
 
-To optionally enforce `system-reserved` on system daemons, specify the parent control group for OS system daemons as the value for `--system-reserved-cgroup` kubelet flag.
+To optionally enforce `system-reserved` on system daemons, specify the parent
+control group for OS system daemons as the value for `--system-reserved-cgroup`
+kubelet flag.
 
-It is recommended that the OS system daemons are placed under a top level control group (`system.slice` on systemd machines for example).
+It is recommended that the OS system daemons are placed under a top level
+control group (`system.slice` on systemd machines for example).
 
-Note that Kubelet **does not** create `--system-reserved-cgroup` if it doesn't exist. Kubelet will fail if an invalid cgroup is specified.
+Note that Kubelet **does not** create `--system-reserved-cgroup` if it doesn't
+exist. Kubelet will fail if an invalid cgroup is specified.
 
 ### Eviction Thresholds
 
 - **Kubelet Flag**: `--eviction-hard=[memory.available<500Mi]`
 
-Memory pressure at the node level leads to System OOMs which affects the entire node and all pods running on it.
-Nodes can go offline temporarily until memory has been reclaimed.
-To avoid (or reduce the probability) system OOMs kubelet provides [`Out of Resource`](./out-of-resource.md) management.
-Evictions are supported for `memory` and `storage` only.
-By reserving some memory via `--eviction-hard` flag, the `kubelet` attempts to `evict` pods whenever memory availability on the node drops below the reserved value.
-Hypothetically, if system daemons did not exist on a node, pods cannot use more than `capacity - eviction-hard`.
-For this reason, resources reserved for evictions are not available for pods.
+Memory pressure at the node level leads to System OOMs which affects the entire
+node and all pods running on it. Nodes can go offline temporarily until memory
+has been reclaimed. To avoid (or reduce the probability) system OOMs kubelet
```

Comment:
s/To avoid (or reduce the probability) system OOMs kubelet.../To avoid (or reduce the probability of) system OOMs, kubelet...